robot:
  wheelbase: 1.10
  track_width: 0.65
  length: 1.40
  width: 0.70
  height: 1.20

  wheel_radius: 0.15
  encoder_resolution: 2048
  drive_type: "ackermann"

imu:
  x: 0.10
  y: 0.0
  z: 0.40
  roll: 0.0
  pitch: 0.0
  yaw: 0.0

gnss:
  x: 0.20
  y: 0.0
  z: 1.00
  roll: 0.0
  pitch: 0.0
  yaw: 0.0

lidar_front:
  x: 0.50
  y: 0.0
  z: 0.50
  roll: 0.0
  pitch: 0.0
  yaw: 0.0

lidar_rear:
  x: -0.50
  y: 0.0
  z: 0.50
  roll: 0.0
  pitch: 0.0
  yaw: 180.0

camera_front:
  x: 0.45
  y: 0.0
  z: 0.80
  roll: 0.0
  pitch: -5.0
  yaw: 0.0

camera_rear:
  x: -0.45
  y: 0.0
  z: 0.80
  roll: 0.0
  pitch: -5.0
  yaw: 180.0
